.gitignore

```diff
@@ -15,3 +15,4 @@
 /libpeerconnection.log
 npm-debug.log
 testem.log
+.vscode
```

Comment:
In _general_ I think editor specific ignores should either be in `.git/info/exclude` (which has the same format as `.gitignore` but isn't checked into the repo) or a global `.gitignore`.